src/components/player.js

```diff
@@ -98,6 +98,48 @@ export default class Player extends BaseObject {
     }
   }
 
+  /**
+   * @typedef {Object} PlaybackConfig
```

Comment:
@towerz I have created a definition of `PlaybackConfig` the way it is already used in HTML5VideoElement - seems we are already inherently using this but it's not actually documented anywhere.

As this could be an "official" API increment, we will be able to drop the support for the deprecated properties in Player options?

Is this the direction we could go?

Reply:
@tchakabam I think it's the right direction. I'm not sure if we can drop the support immediately, but we can add a deprecation notice, just to have a smoother transition.